internal/pkg/public-rpc/get_trace_tree.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 type GetTraceTreeParams struct {
-	TracingHeader string `json:"TracingHeader"`
-	Profile       string `json:"Profile,omitempty"`
-	Region        string `json:"Region,omitempty"`
+	TracingHeader    string `json:"TracingHeader"`
+	Profile          string `json:"Profile,omitempty"`
+	Region           string `json:"Region,omitempty"`
+	FetchChildTraces bool   `json:"FetchChildTraces,omitempty"`
```

Comment:
Should we use `FetchLinkedTraces` instead to name it more consistently with what xray uses in their docs?

Reply:
I think `FetchChildTraces